examples/with-content/src/pages/[...slug].astro

```diff
@@ -0,0 +1,24 @@
+---
+import { getCollection, renderEntry, Collection } from 'astro:content';
+import Layout from '../layouts/Layout.astro';
+
+export async function getStaticPaths() {
+	const blog = await getCollection('blog');
+	return blog.map(post => ({
+		params: { slug: post.slug },
+		props: { post },
+	}))
+}
+
+interface Props {
+	post: Collection<'blog'>;
```

Comment:
Should be `CollectionEntry<'blog'>` now.

I assume this is on your radar from the CI errors, but I’ve noticed that `astro check` now fails if `astro build` or `astro dev` hasn’t been run, and presumably gives stale results if it hasn’t been run sufficiently recently. Ideally `astro check` would generate types.generated.d.ts itself, right?

Reply:
Thanks for the callout! Just fixed 👍